changelogs/unreleased/ghe-643-set-status-after-post-apply.md

```diff
@@ -0,0 +1,3 @@
+## feature/config
+
+* The config status is now only set after the `post_apply` phase (ghe-643).
```

Comment:
Are we need this change in `release/3.0`?

Reply:
```suggestion
* The config status is now set after the `post_apply` phase (ghe-643).
```

Reply:
> Are we need this change in `release/3.0`?

Since this is a bugfix, it makes sense to add this path to `release/3.0`.

@p7nov thanks for the suggestion. Applied.